### ISSUE TITLE
Restart game button badly needs a confirmation dialog

### ISSUE THREAD

**Issue by underyx** [David20321/FTJ#1]
I've just got a game of 30 minutes cleared accidentally after a friend of mine tried to reconnect to the server and hit the wrong button - this, of course, wouldn't be able to happen if the game asked the player if he really wanted to restart the game. Perhaps all players on the servers could be asked if they want to clear the board, making it effectively voting.

**Comment by ghost** [comment]
I agree that the reset button needs a dialogue box as well. I also believe it should be host only.

**Comment by strich** [comment]
Guys I'm not sure the Wolfire devs really intend to continue development on this. I have no confirmation from them to back this up, but I thought I aught to mention it.

**Comment by EhsanKia** [comment]
I wonder if they will accept pull requests from other people though. Let the community keep working on the game?

**Comment by David20321** [comment]
I am still planning to work on it, but it will be in short bursts, kind of like the updates for Receiver.

**Comment by Firetaffer** [comment]
That's cool, so long as you're working on Overgrowth instead >:)
